rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/AdjunctResource.java

```diff
@@ -211,6 +211,11 @@ public Response destroy(String application, String entityToken, String adjunctId
         Entity entity = brooklyn().getEntity(application, entityToken);
         EntityAdjunct adjunct = brooklyn().getAdjunct(entity, adjunctId);
 
+        if (!Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.DELETE_ADJUNCT, adjunct)) {
+            throw WebResourceUtils.forbidden("User '%s' is not authorized to remove adjuncts '%s'",
```

Comment:
Sorry, I should have caught that last time but I think it should say `delete` instead of `remove` for consistency